docker/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM ubuntu:16.04
```

Comment:
Is there a reason to use ubuntu over alpine?

Also, you should really glue up all these RUN statements together, the cleanup needs to happen in the same command (e.g. you could remove the build deps that aren't required later). All of this could be a single RUN command.

Doing that would make this docker file much much smaller.